go.mod

```diff
@@ -3,12 +3,12 @@ module github.com/peak/s5cmd/v2
 go 1.17
 
 require (
-	github.com/aws/aws-sdk-go v1.40.25
+	github.com/aws/aws-sdk-go v1.44.256
```

Comment:
is sdk update necessary for this PR?

Reply:
We have [upgraded](https://github.com/igungor/gofakes3/commit/3f92ea56b09fb4f3d7efec3a40219785ba75dd83) the SDK version in one of our dependencies, which wasn't necessary but I have accidentally bumped it. Now, either we have downgrade the `aws-sdk-go` version in `gofakes3` or keep the minor upgrade I believe.

Reply:
that's ok. i was just curious if the upgrade was mandatory for content-disposition.